src/Model/User/User.php

```diff
@@ -168,14 +168,14 @@ public function setUserType(string $userType): void
         $this->userType = $userType;
     }
 
-    public function getAdditionalProperties(): array
```

Comment:
I would love to have functions like `getProperty(string $propertyName)` etc. for convenience (and stability) here.